src/components/AggregateCriteria/AggregateCriteriaCard.tsx

```diff
@@ -0,0 +1,129 @@
+          }),
+          value: CriteriaAggregateType.NUM_ROWS,
+        },
+        {
+          label: intl.formatMessage({
+            id: 'ui-plugin-bursar-export.bursarExports.aggregate.filter.totalAmount',
+          }),
+          value: CriteriaAggregateType.TOTAL_AMOUNT,
+        },
+      ].sort((a, b) => a.label.localeCompare(b.label)),
+    [intl]
+  );
+
+  return (
+    <Card
+      headerStart={
+        <FormattedMessage id="ui-plugin-bursar-export.bursarExports.aggregate.filter.header" />
+      }
+    >
+      <Row>
+        <Col xs={12} md={4}>
+          <Field
+            name="aggregateFilter.type"
+            defaultValue={CriteriaAggregateType.PASS}
+          >
+            {(fieldProps) => (
+              <Select<CriteriaAggregateType>
+                {...fieldProps}
+                fullWidth
+                marginBottom0
+                required
+                label={
+                  <FormattedMessage id="ui-plugin-bursar-export.bursarExports.aggregate.filter" />
+                }
+                dataOptions={[
+                  {
+                    label: intl.formatMessage({
+                      id: 'ui-plugin-bursar-export.bursarExports.aggregate.filter.none',
+                    }),
+                    value: CriteriaAggregateType.PASS,
+                  },
+                  ...criteriaOptions,
+                ]}
+              />
+            )}
+          </Field>
+        </Col>
+
+        {selectedType !== CriteriaAggregateType.PASS && (
+          <Col xs={12} md={4}>
+            <OperatorSelect name="aggregateFilter.operator" />
+          </Col>
+        )}
+
+        {selectedType === CriteriaAggregateType.NUM_ROWS && (
+          <Col xs={12} md={4}>
+            <Field name="aggregateFilter.amount">
+              {(fieldProps) => (
+                <TextField<number>
+                  {...fieldProps}
+                  fullWidth
+                  marginBottom0
+                  required
+                  type="number"
+                  label={
+                    <FormattedMessage id="ui-plugin-bursar-export.bursarExports.aggregate.filter.numAccounts.amount" />
+                  }
+                  min={1}
+                  step={1}
+                />
+              )}
+            </Field>
+          </Col>
+        )}
+
+        {selectedType === CriteriaAggregateType.TOTAL_AMOUNT && (
+          <Col xs={12} md={4}>
+            <Field name="aggregateFilter.amountDollars">
+              {(fieldProps) => (
+                <TextField<number>
+                  {...fieldProps}
+                  fullWidth
+                  marginBottom0
+                  required
+                  type="number"
+                  label={
+                    <FormattedMessage id="ui-plugin-bursar-export.bursarExports.aggregate.filter.totalAmount.amount" />
+                  }
+                  min={0}
+                  step={0.01}
+                  onBlur={monetaryOnBlur}
```

Comment:
is it your requirement to put 0 on blur when there is no value?  it's a bit strange as if user just touches field value becomes 0.00 and seems like validation will skip it, so unintentionally user can touch it and save 0.00

Reply:
in general it's not fully clear what problem you solve with `monetaryOnBlur`, if you need parse, react-final-form Field has a parse prop where you apply a function - please look how it's solved in other modules

Reply:
The onBlur here is to make it so if we put 3, it makes it 3.00 to make it 3 dollars even. If you want, then we can remove but we thought it was a nice to have.

Reply:
I have removed the blur functionality.